src/pages/EditMonitor.vue

```diff
@@ -246,646 +229,639 @@
                                 <label for="port" class="form-label">{{ $t("Port") }}</label>
                                 <input id="port" v-model="monitor.port" type="number" class="form-control" required min="0" max="65535" step="1">
                             </div>
 
-                            <!-- DNS Resolver Server -->
-                            <!-- For DNS Type -->
-                            <template v-if="monitor.type === 'dns'">
-                                <div class="my-3">
-                                    <label for="dns_resolve_server" class="form-label">{{ $t("Resolver Server") }}</label>
-                                    <input id="dns_resolve_server" v-model="monitor.dns_resolve_server" type="text" class="form-control" :pattern="ipRegex" required>
-                                    <div class="form-text">
-                                        {{ $t("resolverserverDescription") }}
-                                    </div>
-                                </div>
+                            <!-- SNMP Monitor Type -->
+                            <div v-if="monitor.type === 'snmp'" class="my-3">
+                                <label for="snmp_community_string" class="form-label">{{ $t("Community String") }}</label>
+                                <input id="snmp_community_string" v-model="monitor.snmpCommunityString" type="text" class="form-control" required>
+                            </div>
 
-                                <!-- Port -->
-                                <div class="my-3">
-                                    <label for="port" class="form-label">{{ $t("Port") }}</label>
-                                    <input id="port" v-model="monitor.port" type="number" class="form-control" required min="0" max="65535" step="1">
-                                    <div class="form-text">
-                                        {{ $t("dnsPortDescription") }}
+                            <div v-if="monitor.type === 'snmp'" class="my-3">
+                                <label for="snmp_oid" class="form-label">{{ $t("OID (Object Identifier)") }}</label>
+                                <input id="snmp_oid" v-model="monitor.snmpOid" type="text" class="form-control" required>
+                            </div>
+
+                            <div v-if="monitor.type === 'snmp'" class="my-3">
+                                <div class="d-flex align-items-start">
+                                    <div>
+                                        <label for="snmp_condition" class="form-label">{{ $t("Condition") }}</label>
+                                        <select id="snmp_condition" v-model="monitor.snmpCondition" class="form-select me-3">
+                                            <option value=">">></option>
```

Comment:
@CommanderStorm, just to clarify: Are you asking about the compatibility of my PR at its current state, or are you seeking confirmation assuming that #4617 is merged prior to mine (#4717)?

After reviewing #4617, I can confirm that my PR (#4717) remains compatible independently of #4617. This is because I am introducing new inputs tailored to the SNMP monitor, which will house my conditional string (`monitor.snmpCondition`) and the compared value (`monitor.snmpControlValue`).